frontend/src/app/containers/App.js

```diff
@@ -189,16 +207,14 @@ function sendToGA(type, dataIn) {
 
 const mapStateToProps = state => ({
```

Comment:
At some point soon we should split up this monolithic function and have one for every container.

Not in this pull request though.

Reply:
Maybe file an issue toward that? I think the majority of these are shared across containers, so that could use some homework to figure out how true that might be

Reply:
Filed:

https://github.com/mozilla/testpilot/issues/2924

Maybe it won't be worth it, but we should at least investigate it.